### ISSUE TITLE
[Bug]: [cn-dn with flushing] cn do not send delete operation when updating auto incr table

### ISSUE THREAD

**Issue by aptend** [matrixorigin/matrixone#5990]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

```
mysql> create account NtrWE02sgtvCXu admin_name '謳95071驊2连5202曹' identified by '鑊78狿殻01948楲777';
ERROR 20305 (HY000): tae data: duplicate
mysql> 
```

### Steps to Reproduce

```Markdown
start: ./mo-service -launch etc/launch-flush-tae-CN-tae-DN/launch.toml

with smaller flush config like 

[dn.Ckp]
scanner-interval = "100ms"
execution-levels = 2
execution-interval = "10ms"
catalog-ckp-interval = "10ms"
catalog-unckp-limit = 1
flush-interval = "100ms"


create account NtrWE02sgtvCXu admin_name '謳95071驊2连5202曹' identified by '鑊78狿殻01948楲777';
```

**Comment by aptend** [comment]
partitionDeleteBatch returns empty batch and doesn't send delete operation to Precommit interface

**Comment by nnsgmsone** [comment]
fixed by pr #6003
